packages/code-studio/src/main/AppRouter.tsx

```diff
@@ -1,13 +1,19 @@
 import React, { ReactElement } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import {
+  BrowserRouter as Router,
+  Route,
+  Redirect,
+  Switch,
+} from 'react-router-dom';
 import AppInit from './AppInit';
 
 function AppRouter(): ReactElement {
   return (
     <Router basename={import.meta.env.BASE_URL}>
       <Switch>
+        <Route path="/" component={AppInit} />
         <Route path="/notebook/:notebookPath+" component={AppInit} />
-        <Route path="*" component={AppInit} />
+        <Redirect from="*" to="/" />
```

Comment:
I think this was initially just a wildcard path instead of a redirect for DHE deep linking that didn't require us to put a specific route in the router for each deep linking route

That said, it's probably better to have a route listed for each deep linking route in DHE because it gives us a more centralized location for them. Easier discoverability

Reply:
I added the wildcard path with a previous change: https://github.com/deephaven/web-client-ui/pull/1204/commits/d51bd7d96963bee2969415da1d39c556f2bc3b16
But that no longer applies, so I'm switching this back.